gulp-tasks/utils/rollup-helper.js

```diff
@@ -14,27 +13,20 @@ module.exports = {
     const plugins = [];
 
     const babelConfig = {
-      presets: [['env', {
+      presets: [['@babel/preset-env', {
         targets: {
-          // This corresponds to the version of Chroumium used by
-          // Samsung Internet 5.x, which is the minimum non-evergreen browser
+          // This corresponds to the version of Chromium used by
+          // Samsung Internet 6.x, which is the minimum non-evergreen browser
           // we currently support.
-          browsers: ['chrome >= 51'],
+          browsers: ['chrome >= 56'],
         },
-        modules: false,
       }]],
-      // This will result in references to `babelHelpers.asyncToGenerators()`
-      // in the transpiled code. The declaration of this method lives in the
-      // external-helpers.js file and is injected by Rollup once into the
-      // workbox-core bundle.
-      plugins: ['external-helpers'],
-      externalHelpers: true,
     };
     plugins.push(babel(babelConfig));
 
     let minifyBuild = buildType === constants.BUILD_TYPES.prod;
     if (minifyBuild) {
-      const uglifyOptions = {
+      const terserOptions = {
         mangle: {
           properties: {
```

Comment:
I discovered recently that we probably need to enable the option to make property mangling consistent across files (when we're outputting multiple files). I can file an issue to do that in a separate PR.